src/utils/authentication.py

```diff
@@ -0,0 +1,13 @@
+import bcrypt
+
+from src.cfg.settings import security
+
+TOKEN = security["token"]
+hashed = bcrypt.hashpw(bytes(TOKEN, "utf-8"), bcrypt.gensalt())
+
+
+def verify_password(plain_password):
+    if bcrypt.checkpw(str.encode(plain_password), hashed):
```

Comment:
for security purposes you might want to make sure input is only a certain length and not empty

Reply:
Fixes Implemented